queries.sql

```diff
@@ -23,3 +23,40 @@ WHERE name NOT LIKE 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg >= 10.40 AND weight_kg <= 17.30;
+
+/* Begin of a transaction */
+BEGIN;
+
```

Comment:
- [ ] Please from the project requirement, 
  - Inside a transaction:
    - Delete all animals born after Jan 1st, 2022.
    - Create a savepoint for the transaction.
    - Update all animals' weight to be their weight multiplied by -1.
    - Rollback to the savepoint
    - Update all animals' weights that are negative to be their weight multiplied by -1.
    - Commit transaction

Please make sure you do that and **Take a screenshot of the results of your actions.** :pray: